### DISCUSSION TITLE
Show all bundles in the graph

### DISCUSSION THREAD

**Discussion by jodaka** [btd/rollup-plugin-visualizer#87] [Q&A] [answered]
My rollup build generates 3 different JS bundle. One per entry file. Visualizer seems to render only the last bundle. 

In my project I have one entry file, but it uses dynamic imports, so there are multiple generated JS files. Yet visualizer only shows JS bundle for entry point, ignoring all the others. 

Is it intentional? I couldn't find option to make visualizer show all bundles (or show some dropdown allowing to choose bundle)

Seems like my question is related to https://github.com/btd/rollup-plugin-visualizer/issues/75

**Comment by btd** [comment, marked as answer]
I need example of you config.

**Reply by jodaka** [reply]
Please disregard my question. I figured out I had a problem with config. My rollup.config returned an array of config options for each entry. While in fact I needed to return single options object with an array in 'input' property. 

I just tested both of my project. 
One that generates 2 chunks due to dynamic imports: 
![image](https://user-images.githubusercontent.com/83644/111107383-cd742a00-8567-11eb-8318-07b0a2046246.png)
Two chunks are visible. 

And another with 3 different entry points — both are rendered correctly in visualizer. 
![image](https://user-images.githubusercontent.com/83644/111107530-09a78a80-8568-11eb-8c75-1a6e6860645a.png)
generates 4 chucks (one with shared code and one per each entry point)
